react-components/storybook-static/index.json

```diff
@@ -0,0 +1 @@
Main","title":"Example/PrimitiveWrappers/Image360CollectionContainer","importPath":"./stories/Image360CollectionContainer.stories.tsx","tags":["autodocs","story"]},"example-image360details--main":{"type":"story","id":"example-image360details--main","name":"Main","title":"Example/Image360Details","importPath":"./stories/Image360Details.stories.tsx","tags":["autodocs","story"]},"example-toolbar-layerscontainer--main":{"type":"story","id":"example-toolbar-layerscontainer--main","name":"Main","title":"Example/Toolbar/LayersContainer","importPath":"./stories/LayersContainer.stories.tsx","tags":["autodocs","story"]},"example-modelsstyling--main":{"type":"story","id":"example-modelsstyling--main","name":"Main","title":"Example/ModelsStyling","importPath":"./stories/ModelsStyling.stories.tsx","tags":["autodocs","story"]},"example-primitivewrappers-pointcloudcontainer--main":{"type":"story","id":"example-primitivewrappers-pointcloudcontainer--main","name":"Main","title":"Example/PrimitiveWrappers/PointCloudContainer","importPath":"./stories/PointCloudContainer.stories.tsx","tags":["autodocs","story"]},"example-reveal3dresources--main":{"type":"story","id":"example-reveal3dresources--main","name":"Main","title":"Example/Reveal3DResources","importPath":"./stories/Reveal3DResources.stories.tsx","tags":["autodocs","story"]},"example-revealkeepalive--main":{"type":"story","id":"example-revealkeepalive--main","name":"Main","title":"Example/RevealKeepAlive","importPath":"./stories/RevealKeepAlive.stories.tsx","tags":["autodocs","story"]},"example-rulebasedcoloring--main":{"type":"story","id":"example-rulebasedcoloring--main","name":"Main","title":"Example/RuleBasedColoring","importPath":"./stories/RuleBasedColoring.stories.tsx","tags":["autodocs","story"]},"example-primitivewrappers-scenecontainer--main":{"type":"story","id":"example-primitivewrappers-scenecontainer--main","name":"Main","title":"Example/PrimitiveWrappers/SceneContainer","importPath":"./stories/SceneContainer.stories.tsx","tags":["autodocs","story"]},"example-searchhooks--main":{"type":"story","id":"example-searchhooks--main","name":"Main","title":"Example/SearchHooks","importPath":"./stories/SearchHooks.stories.tsx","tags":["autodocs","story"]},"example-splitwindow--main":{"type":"story","id":"example-splitwindow--main","name":"Main","title":"Example/SplitWindow","importPath":"./stories/SplitWindow.stories.tsx","tags":["autodocs","story"]},"example-toolbar--main":{"type":"story","id":"example-toolbar--main","name":"Main","title":"Example/Toolbar","importPath":"./stories/Toolbar.stories.tsx","tags":["autodocs","story"]},"example-topbar--main":{"type":"story","id":"example-topbar--main","name":"Main","title":"Example/Topbar","importPath":"./stories/Topbar.stories.tsx","tags":["autodocs","story"]},"example-vieweranchor--main":{"type":"story","id":"example-vieweranchor--main","name":"Main","title":"Example/ViewerAnchor","importPath":"./stories/ViewerAnchor.stories.tsx","tags":["autodocs","story"]}}}
```

Comment:
Should this be part of the PR?

Reply:
yes, it replaces the stories.json (which you will notice has been deleted). I use if for type inference when loading a story for visual tests. See loadStory.ts.